app/app.go

```diff
@@ -149,6 +150,7 @@ func (app *kaspadApp) main(startedChan chan<- struct{}) error {
 			log.Criticalf("Graceful shutdown timed out %s. Terminating...", shutdownTimeout)
 		}
 		log.Infof("Kaspad shutdown complete")
+		logger.BackendLog.Close()
```

Comment:
It's apparently not quite clear where is the last exit route of the app. 
I thought this ism but it looks like some goroutines are still exiting and writing logs after this is called?